roles/elastic-stack/ansible-kibana/tasks/main.yml

```diff
@@ -161,10 +161,15 @@
     validate_certs: no
     status_code: 200, 404
 
+- name: Create wazuh plugin config directory
```

Comment:
Playbook executes as root but so I assume the folder will be root:root, but it should be root:kibana right? Please let me know if it has right owner and group.

Reply:
Done in 
9cdce6e